hyundai_kia_connect_api/KiaUvoApiEU.py

```diff
@@ -406,7 +406,20 @@ def _update_vehicle_properties_ccs2(self, vehicle: Vehicle, state: dict) -> None
 
         vehicle.engine_is_running = get_child_value(state, "DrivingReady")
 
-        # TODO: vehicle.air_temperature = get_child_value(state, "Cabin.HVAC.Driver.Temperature.Value")
+        if get_child_value(state, "Cabin.HVAC.Row1.Driver.Temperature.Value"):
+            tempIndex = get_hex_temp_into_index(
```

Comment:
To confirm is your raw data in hex? I got the impression it wasn't. This is only needed for cars that don't show the decimal value.

Reply:
You are correct, it is decimal, just found also that when the car is off the value is set to "OFF"

Reply:
If the value is off we will need to set to that a number as the data type isn't a string.